Assets/MixedRealityToolkit/_Core/Utilities/Lines/DataProviders/ParabolaPhysicalLineDataProvider.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Attributes;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Utilities.Lines.DataProviders
+{
+    /// <summary>
+    /// Creates a parabolic line based on physics.
+    /// </summary>
+    public class ParabolaPhysicalLineDataProvider : ParabolaLineDataProvider
+    {
+        [SerializeField]
+        [Vector3Range(-1f, 1f)]
+        private Vector3 direction = Vector3.forward;
+
+        public Vector3 Direction
+        {
+            get { return direction; }
+            set
+            {
+                direction.x = Mathf.Clamp(value.x, -1f, 1f);
+                direction.y = Mathf.Clamp(value.y, -1f, 1f);
+                direction.z = Mathf.Clamp(value.z, -1f, 1f);
+            }
+        }
+
+        [SerializeField]
+        private float velocity = 2f;
+
+        public float Velocity
+        {
+            get { return velocity; }
+            set { velocity = value; }
+        }
+
+        [SerializeField]
+        private float timeMultiplier = 1f;
+
+        public float TimeMultiplier
+        {
+            get { return timeMultiplier; }
+            set { timeMultiplier = value; }
+        }
+
+        [SerializeField]
+        private bool useCustomGravity = false;
+
+        public bool UseCustomGravity
+        {
+            get { return useCustomGravity; }
+            set { useCustomGravity = value; }
+        }
+
+        [SerializeField]
+        private Vector3 gravity = Vector3.down * 9.8f;
```

Comment:
tooltip. range?

Reply:
I don't think we should add a range there, in case the user wants to put in a specific amount.